src/pages/SubmissionConfig.js

```diff
@@ -26,8 +33,20 @@ export const SubmissionConfig = ({ methodId }) => {
   const [runSetName, setRunSetName] = useState()
   const [runSetDescription, setRunSetDescription] = useState()
 
+  // TODO: this should probably be moved to a scope more local to the data selector
+  const [sort, setSort] = useState({ field: 'name', direction: 'asc' })
+
   const signal = useCancellation()
 
+  const loadRecordsData = async recordType => {
```

Comment:
IMO, all of these `load<X>Data` functions should be outside of the `useOnMount` scope. I had that change here originally, but it made the diff harder to interpret